patches/react-native-animateable-text+0.16.0-beta.0.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/node_modules/react-native-animateable-text/lib/typescript/src/index.d.ts b/node_modules/react-native-animateable-text/lib/typescript/src/index.d.ts
+index bbcd336..8e962f1 100644
+--- a/node_modules/react-native-animateable-text/lib/typescript/src/index.d.ts
++++ b/node_modules/react-native-animateable-text/lib/typescript/src/index.d.ts
+@@ -1,4 +1,4 @@
+-declare const AnimateableText: import("react").FunctionComponent<import("react-native-reanimated").AnimateProps<Omit<import("react-native").TextProps, "children"> & {
++declare const AnimateableText: import("react").FunctionComponent<import("react-native-reanimated").AnimateProps<import("react-native").TextProps & {
```

Comment:
just curious - why do we need to stop removing children type?

Reply:
If I remember correctly this is due to a type change in React types that no longer automatically adds children. If we omit children here then the resulting type no longer had it.